packages/bpk-component-router-link/stories.js

```diff
@@ -19,17 +19,20 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from 'bpk-storybook-utils';
+import { BrowserRouter } from 'react-router-dom';
 
 import BpkRouterLink from './index';
 
 storiesOf('bpk-component-router-link', module).add('Example', () => (
-  <div>
-    <BpkRouterLink to="#" onClick={action('#1 clicked')}>
-      Link #1
-    </BpkRouterLink>
-    <br />
-    <BpkRouterLink to="#" onClick={action('#2 clicked')}>
-      Link #2
-    </BpkRouterLink>
-  </div>
+  <BrowserRouter>
```

Comment:
To get rid of an error in this version about things needing to be inside a `Router`.